examples/tree/plot_unveil_tree_structure.py

```diff
@@ -122,24 +123,35 @@
                 feature=feature[i],
                 threshold=threshold[i],
                 right=children_right[i],
-                value=values[i],
+                value=np.around(values[i], 3),
             )
         )
 
 # %%
 # What is the values array used here?
 # -----------------------------------
 # The `tree_.value` array is a 3D array of shape
-# [``n_nodes``, ``n_classes``, ``n_outputs``] which provides the count of samples
-# reaching a node for each class and for each output. Each node has a ``value``
-# array which is the number of weighted samples reaching this
-# node for each output and class.
+# [``n_nodes``, ``n_classes``, ``n_outputs``] which provides the proportion of samples
+# reaching a node for each class and for each output.
+# Each node has a ``value`` array which is the proportion of weighted samples reaching
+# this node for each output and class with respect to the parent node.
+#
+# One could convert this to the absolute weighted number of samples reaching a node,
+# by multiplying this number by `tree_.weighted_n_node_samples[node_idx]` for the
+# given node, or by `tree_.n_node_samples[node_idx]` for the unweighted number of
```

Comment:
I am a bit unsure about multiplying `tree_.value` (proportion of samples taking into account sample weights) by `tree_.n_node_samples[node_idx]` which does not take sample weights into account. It seems to me that the output of this multiplication has not a clear meaning, but maybe I am missing something ...

Maybe only mention `tree_.weighted_n_nodes_sample` and mention that sample weights are not used in this example, which is the same as being one for all the samples?

Reply:
Done: [3c1f24a](https://github.com/scikit-learn/scikit-learn/pull/29331/commits/3c1f24a51e0d989ca261168716a208b4c5d186f6)